### ISSUE TITLE
How to make Inventory Tweaks' Item sorting work with DE Draconium chest 

### ISSUE THREAD

**Issue by GoldenTGraham** [Inventory-Tweaks/inventory-tweaks#486]
I've looked for a solution and asked on Reddit but I've not found anything that can help me. Any and all help is much appreciated

(Also posted this on the Draconic Evolution Github)

**Comment by llloyd4** [comment]
Yeah, I middle click sort and it pulls everything out of their respected slots into the center so I got to go back and put the core back into the core slot, gets to be really annoying.>.<
